pypeit/scripts/show_2dspec.py

```diff
@@ -133,7 +133,15 @@ def main(args):
     slit_ids = [edgetrace.get_slitid(shape, tslits_dict['slit_left'], tslits_dict['slit_righ'],
                                      ii)[0] for ii in range(tslits_dict['slit_left'].shape[1])]
     # Show the bitmask?
-    mask_in = mask if args.showmask else None
+    if args.showmask:
+        mask_in = mask
+        # Unpack the bitmask
+        bitMask = ImageBitMask()
+        bpm, crmask, satmask, minmask, offslitmask, nanmask, ivar0mask, ivarnanmask, extractmask \
```

Comment:
This unpack is not needed, right?

Reply:
good eye.  Am just commenting out for now.